public/pages/Transforms/utils/metadataHelper.tsx

```diff
@@ -67,7 +69,7 @@ export const renderStatus = (metadata: TransformMetadata | undefined): JSX.Eleme
   }
 
   return (
-    <EuiFlexGroup gutterSize="xs">
+    <EuiFlexGroup gutterSize="xs" alignItems={"center"}>
```

Comment:
Also here: `alignItems= {"center"}` -> `alignItems= "center"`

Reply:
Same as above. I'll clean that up in the next revision. I believe the IDE added the curly brackets automatically.